### PR TITLE
Remove unused org.kohsuke.stapler:stapler-adjunct-zeroclipboard

### PR DESCRIPTION
**Pull request by jglick**
Cleans up a dep made obsolete in #3780. `zeroclipboard` is a flash implementation of the Copy-to-Clipboard control

### Proposed changelog entries

* Removing the deprecated `zeroclipboard` UI component and the unused dependency on ZeroClipboard

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by jvz — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
FTR https://groups.google.com/g/jenkinsci-dev/c/gX0Avxn7sxw for the dev list proposal

**Comment by jglick** [comment]
I do not think this is technically `internal`, as it does remove a dependency. While we do not _know_ of any plugins directly using the Stapler adjunct for ZC, in principle there could be one, and it would be broken by this change.

**Review by res0nance — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
We have 'developer' for that

**Comment by jglick** [comment]
Well, yes, but we do not actually _expect_ any developers to notice at all. (Otherwise why would we bother with separate `developer` and `plugin-api-changes` labels?)

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by oleg-nenashev** [comment]
Does it require upgrade guidelines for developers, just in case? I can imagine a proprietary plugin using the control though it is highly unlikely

**Comment by oleg-nenashev** [comment]
> Well, yes, but we do not actually expect any developers to notice at all.

Well, there is a taglib form control in https://github.com/stapler/stapler-adjunct-zeroclipboard/blob/master/src/main/resources/org/kohsuke/stapler/zeroclipboard.jelly . Arguably, this is a public API accessible to developers. I updated the changelog suggestion accordingly @jglick , PTAL and revert if needed

**Pull request merged**
